### DISCUSSION TITLE
Route [filament.pages.dashboard] not defined.

### DISCUSSION THREAD

**Discussion by aksalsf** [filamentphp/filament#3199] [Help]
I got Route [filament.pages.dashboard] not defined. when login

after running
```
php artisan vendor:publish --tag=filament-config
```

```
php artisan vendor:publish --tag=filament-views
```

How to solve it?

I follow https://www.codef.site/blog/laravel/how-to-customize-laravel-filament-login-page

**Comment by danharrin** [comment]
Please don't publish the views, delete that folder... try `php artisan filament:upgrade` to clear your view cache

**Reply by aksalsf** [reply]
Noted. But, how to custom the existing page like login page or like creating register page which is doesn't need dashboard layout?

**Reply by danharrin** [reply]
If you need registration, use https://github.com/jeffgreco13/filament-breezy

**Comment by sagar03d** [comment]
but to solve this error? cache clear not working

**Reply by brunocascio** [reply]
If you're using socialite plugin, you can do it in this way:

```php
FilamentSocialitePlugin::make()
  // here you can define your dashboard (or non-dash) route
  ->setDashboardRouteName('filament.dashboard.resources.projects.index')
  ->setProviders([
    ...
  ])
```

**Comment by SanderMuller** [comment]
Try running `php artisan filament:assets`
